Cue/app/api/Library.js

```diff
@@ -1,19 +1,58 @@
 // @flow
-import CueApi from './CueApi';
+import CueApi from './CueApi'
+import DeviceInfo from 'react-native-device-info'
+
 
 var LibraryApi = {
 
-	fetchLibrary() {
-		let endpoint = '/api/v1/library';
-		console.info('Fetching Library from endpoint: ' + endpoint)
-		return CueApi.fetch(endpoint);
-	},
-
-	fetchDeck(uuid : string) {
-		let endpoint = '/api/v1/deck/' + uuid;
-		console.info('Fetching deck "' + uuid + '" from endpoint: ' + endpoint)
-		return CueApi.fetch(endpoint);
-	},
+  fetchLibrary() {
+    let endpoint = '/api/v1/library';
+    console.info('Fetching Library from endpoint: ' + endpoint)
+    return CueApi.fetch(endpoint);
+  },
+
+  fetchDeck(uuid : string) {
+    let endpoint = '/api/v1/deck/' + uuid;
+    console.info('Fetching deck from endpoint: ' + endpoint)
+    return CueApi.fetch(endpoint);
+  },
+
+  createDeck(
+      name: string,
+      tags: Array<string> = [],
+      cards: Array<{front:string; back:string}> = []) {
+    let endpoint = '/api/v1/deck/add';
+
+    let body = JSON.stringify ({
+      name,
+      tags: tags,
+      device: DeviceInfo.getUniqueID(),
+      cards: cards,
+    })
+		console.info('Creating deck "' + name + '" at endpoint ' + endpoint)
+    return CueApi.fetch(endpoint, 'POST', body)
+  },
+
+  editDeck (change){
+    let endpoint = 'api/v1/deck/' + change.uuid
+    let body = JSON.stringify({
+      device: DeviceInfo.getUniqueID(),
```

Comment:
Should we have a user-facing "friendly name" for devices for the purposes of making merge conflicts easier on users? @ClaytonPassmore @henryc132